NodeJs-API/routes/router.js

```diff
@@ -142,14 +146,62 @@ router.use('/spotify', async function (req, res) {
     }
 });
 
+// router.get('/playlist/:name/:desc', function (req, res) {
+//     if (typeof req.params.name !== 'undefined' && typeof req.params.desc !== 'undefined') {
+//         var nameOfPlaylist = req.params.name;
+//         var descOfPlaylist = req.params.desc;
+//         if (req.session.spotifyID === undefined) {
+//             res.send('Spotify Id missing. Call /me');
+//         } else if (req.session.loggedin) {
+//             Playlist.createPlaylist(req.session.spotifyID, req.session.auth_token, nameOfPlaylist, descOfPlaylist);
+//             res.end();
+//         } else {
+
+//             res.send('Please login first');
+//         }
+//     }
+// });
+
+//artistId:44TGR1CzjKBxSHsSEy7bi9 //for testing
+//country:NL
+router.get('/playlist/:name/:description/:artistId/:country', async function (req, res) {
```

Comment:
i prefer not to have the name and the description of the playlist removed from this. but if you don't want to remove this, let me know how you want to call in the FE